src/libres/lib/include/ert/enkf/enkf_main.hpp

```diff
@@ -118,21 +103,15 @@ enkf_main_load_from_run_context(enkf_main_type *enkf_main,
                                     std::vector<bool> active_mask,
                                     enkf_fs_type *sim_fs,
                                     std::vector<run_arg_type*> run_args);
-
+char *enkf_main_read_alloc_current_case_name(const enkf_main_type *enkf_main);
```

Comment:
Seems like this is only changing location in the diff.

Reply:
Yes, moved it back :+1: